packages/core/src/amazonqFeatureDev/constants.ts

```diff
@@ -15,7 +15,7 @@ export const featureDevChat = 'featureDevChat'
 export const featureName = 'Amazon Q Developer Agent for software development'
 
 // Max allowed size for file collection
-export const maxRepoSizeBytes = 200 * 1024 * 1024
+export const maxRepoSizeBytes = 250 * 1024 * 1024
```

Comment:
Why are we changing this ?

Reply:
This shouldn't be committed, removing it from the CR